pages/profile.jsx

```diff
@@ -85,13 +85,22 @@ class Profile extends React.Component {
           <Bio {...userProfile} user={this.state.user} history={this.props.history} />
         </div>
       </div>
-      <hr />
-      <ProfileTabGroup
-        profileId={userProfile.profile_id}
-        myProfile={userProfile.my_profile}
-        entryCount={userProfile.entry_count}
-        activeTab={activeTab}
-      />
+      <div className="row">
+        <div className="col-12">
+          <hr />
```

Comment:
Does this `hr` really need it's own row? Could it just go right before line 95?

Reply:
good point. will update!